README.md

```diff
@@ -23,7 +23,7 @@ Attestation is the system by which something produces evidence about itself that
 ## What part does Veraison play in this?
 Given the above, it can be challenging to build just one Verification Service solution which can address all deployments for a technology that needs to produce Attestation reports to prove its trustworthiness. If that then implies that each deployment needs a custom service, there is a significant software barrier and hence cost of entry to establishing a system that can be used in a secure manner. Veraison aims to provide consistency and convenience to solving this problem by building the software components that can be used to build Attestation Verification Services. The Veraison project will also build some reference implementations in order to prove the integration principles of those components and to provide a convenient starting point for some implementations.
 
-For further details, see the work in progress documentation under [docs](docs)
+For further details, see the work in progress documentation under [docs](https://github.com/veraison/docs/README.md).
```

Comment:
As now the documentation is not in implicit docs folder inside Veraison, it is better to have a separate bullet ( ## Veraison Documentation) and then link it. This just makes it more explicit details about the project documentation!

Reply:
ok, I will add a subsection below